app/Exceptions/Handler.php

```diff
@@ -45,6 +45,9 @@ public function report(Exception $e)
      */
     public function render($request, Exception $e)
     {
+        if ($e instanceof ModelNotFoundException){
+            return response('Invalid Id!!!');
+        }
```

Comment:
rollback these changes, and simply wrap your Eloquent methods in try/catch blocks - they will throw the exception and you will handle them accordingly - no need to fiddle with these settings